script/functions.sh

```diff
@@ -12,10 +12,8 @@ SPECS_HAVE_RUN_FILE=specs.out
 MAINTENANCE_BRANCH=`cat maintenance-branch`
 
 # Don't allow rubygems to pollute what's loaded. Also, things boot faster
-# without the extra load time of rubygems. Only works on MRI Ruby 1.9+
-if is_mri_192_plus; then
-  export RUBYOPT="--disable=gem"
-fi
+# without the extra load time of rubygems.
+export RUBYOPT="--disable=gem"
```

Comment:
Should we remove it too? Because I read "Only works on MRI Ruby 1.9+"? 🤔

Reply:
Also those change should come from rspec-dev no?

Reply:
1.9+ means 2.3-2.7 and above, so let's keep it for now. I know it went problematic for `rspec-rails`, but probably will still work for other repos.

On a funny note `rspec-core` had this in `spec/spec_helper.rb:
```ruby
require 'rubygems' if RUBY_VERSION.to_f < 1.9
```

Reply:
>  Also those change should come from rspec-dev no?

Yes, absolutely. But I'm experimenting here of what can be stripped off, so we can find what needs to remain because it's still used in some of the repos and what can be removed in `rspec-dev`. I'll extract this and will send a follow-up PR to `rspec-dev`and to other repos against `4-0-dev`.